From e0b0ea90ecc0dbefc6aef2650e88ba88260935b9 Mon Sep 17 00:00:00 2001
From: Tom Rix <trix@redhat.com>
Date: Fri, 29 Sep 2023 17:21:13 -0700
Subject: [PATCH] disable submodule search

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0fd886d945..e397df8fb6 100644
--- a/setup.py
+++ b/setup.py
@@ -458,7 +458,7 @@ def mirror_files_into_torchgen():
 def build_deps():
     report("-- Building version " + version)
 
-    check_submodules()
+    # check_submodules()
     check_pydep("yaml", "pyyaml")
 
     build_caffe2(
-- 
2.43.0

